R/plan.R

```diff
@@ -24,25 +24,25 @@ rnaseq_plan <- function(metadata_id, metadata_version, counts_id,
                         biomart_id, biomart_version, host, filters,
                         organism){
   drake::drake_plan(
-    import_metadata = get_data(!!metadata_id,
+    import_metadata = sageseqr::get_data(!!metadata_id,
```

Comment:
It shouldn't be necessary to prefix these function calls with `sageseqr::`, was there something not working that prompted this addition?

Reply:
It was complaining that it could not find `sageseqr::cqn` but I believe it was due to how I ordered the @export calls with `cqn`, `mclust` and `quantreg`; I fixed this before I created the PR. 

I tested removing this prefix with `devtools::check()` and did not pass a warning, so I removed them.